### PR TITLE
Chmod only shell scripts

### PR DESCRIPTION
**Pull request by billpratt**

### CODE REVIEW

File: Deployment.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Open a bash shell and execute [inception.sh](deployment/inception.sh)
 
 ```bash
 cd /deployment
-chmod 775 *
```
Comment (wesyao) [inline Deployment.md]: shouldn't this be `555`?

Comment (billpratt) [reply Deployment.md]: You don't want people to be able to write? 7=wrx 5 = r-x

**Comment by billpratt** [comment]
@aaron-schnieder Since you wrote this, any reason why we couldn't merge this?

**Comment by aaron-schnieder** [comment]
@billpratt  LGTM - merging.

**Pull request merged**